nservicebus/messaging/message-identity.md

```diff
@@ -5,10 +5,12 @@ component: Core
 ---
 
 
-Each message that is dispatched from an [Endpoint](/nservicebus/endpoints/) has a unique identity. NServiceBus generates and stores this value as a [header](/nservicebus/messaging/headers.md) on the message named `NServiceBus.MessageId`.
+Each message dispatched from an [Endpoint](/nservicebus/endpoints/) has a unique identity. NServiceBus generates and stores this value as a [header](/nservicebus/messaging/headers.md) on the message named `NServiceBus.MessageId`.
 
 Many features take advantage of message identity. For example, the [Outbox](/nservicebus/outbox) uses message identity to deduplicate messages and [recoverability](/nservicebus/recoverability/) uses message identity to keep track of how many times the endpoint has tried to process a message.
 
+> [!WARNING]
```

Comment:
What actionable thing does the user derive from this warning and why is it a warning and not an information level thing? I would rephrase it to something like

Reply:
I like the proposal